Explorer/src/components/containerTable.tsx

```diff
@@ -88,10 +86,10 @@ export default function ContainerTable({
             <span>
               <Dropdown inline label="STATUS" dismissOnClick={true}>
                 <Dropdown.Item>
-                  <a onClick={() => handleSortAsc()}>Assending</a>
+                  <button onClick={() => handleSortAsc()}>Ascending</button>
```

Comment:
I just realized that moving
`onClick={() => handleSortAsc()`
up to <Dropdown.Item> makes the whole Dropdown.Item clickable